rosdep/python.yaml

```diff
@@ -2700,6 +2700,14 @@ python-mako:
 python-mapnik:
   debian: [python-mapnik]
   ubuntu: [python-mapnik]
+python-marisa:
```

Comment:
Do you need a python2 version? If not we'd rather not clutter up the rules with an unused key.

Reply:
Yes, we still have systems running Melodic